ReleaseHistory.md

```diff
@@ -16,6 +16,7 @@
 - NEW => new feature 
 
 ## UNRELEASED
+* DEP: Update `Sarif.Sdk` submodule from [bc8cb57 to fd6e615](https://github.com/microsoft/sarif-sdk/compare/bc8cb57...fd6e615). Reference full [SARIF SDK Release History](https://github.com/microsoft/sarif-sdk/blob/fd6e615/ReleaseHistory.md).
 
```

Comment:
This line is inconsistent with other entries in this file: they all need to be consistent. I think Stacy recommended removing the word `Full`, as below:

if that is correct, please update the entire release history to bring all of these DEP entries into conformance. Also someone needs to make sure our release notes maintenance documentation has the correct standard listed there.

> * DEP: Update `Sarif.Sdk` submodule from [bc8cb57 to fd6e615](https://github.com/microsoft/sarif-sdk/compare/bc8cb57...fd6e615). [SARIF SDK Release History](https://github.com/microsoft/sarif-sdk/blob/fd6e615/ReleaseHistory.md).

Reply:
Also, why exactly did you need to make this dependency update? It is a good practice to cite the specific incoming change that's critical to binskim.